scripts/histmakers/mw_with_mu_eta_pt.py

```diff
@@ -134,48 +159,85 @@
     from wremnants import recoil_tools
     recoilHelper = recoil_tools.Recoil("highPU", args, flavor="mu")
 
-smearing_weights_procs = []
-#graph building for Wsample with helicity weights
-def whistosbyHelicity(df, results, dataset, reco_sel_GF, era):
-    #list(range(0,50,5)).append(np.inf) ,
-    axis_ptVgen = hist.axis.Variable(
-        [0., 5., 10., 15., 20., 25., 30., 35., 40., 45., 50.],
-        name = "ptVgen", underflow=False, overflow=False
-    )
-    #axis_ptVgen.append(np.inf)
-    #Taken from w-z gen histomaker     
-    axis_absYVgen = hist.axis.Variable(
-        #[0, 0.25, 0.5, 0.75, 1, 1.25, 1.5, 1.75, 2, 2.25, 2.5, 2.75, 3, 3.25, 3.5, 3.75, 4., 5., np.inf], 
-        [0, 0.25, 0.5, 0.75, 1., 1.5, 2.5, 5.],
-        name = "absYVgen", underflow=False, overflow=False
-    )
-    print('Entered function whistobyHelicity')                
-    nominal_cols = ["goodMuons_eta0", "goodMuons_pt0", "goodMuons_charge0", "passIso", "passMT", "absYVgen", "ptVgen"]
-    nominal_axes = [axis_eta, axis_pt, axis_charge, axis_passIso, axis_passMT, axis_absYVgen, axis_ptVgen]
+######################################################
+######################################################
+######################################################
+## FIXME:
+## move next functions below to some other place to import from
```

Comment:
Yes, that was the plan. Do you want it done already in this PR though? These restricting things may take time, presumably a new PR would be preferred, since most likely I should touch other things and it would spoil the current review already made by other people

Reply:
I think it would be better to already now move them into a separate function, would also possibly better avoid future merge conflict since then you would only change the theoryAgnostic file and not the main histmaker